### PR TITLE
Attempt to fix off-by-one error

### PR DESCRIPTION
**Pull request by maxwofford**
This reverts commit 025b2bb7f501fb0cc4eaf874d1a2f0e4a11d9170.

### CODE REVIEW

**Comment by zachlatta** [comment]
Oh no, why? Can we get this back in there? I hate having to manually update that value.

**Comment by maxwofford** [comment]
@zachlatta: that was a miscommit -- working on finding the cause of the off-by-one issue we've been having. Won't merge anything that removes unrelated features like the automatic city count

**Comment by zachlatta** [comment]
❤️

--
Zach Latta

https://zachlatta.com ( http://zachlatta.com )
@zachlatta ( https://twitter.com/zachlatta ) / fb ( https://www.facebook.com/crynix ) / github ( https://github.com/zachlatta )

File: src/components/EventCard.js [hunk 1]
```diff
@@ -141,7 +162,6 @@ export default ({
           <Image
             itemProp="image"
             src={logo}
-            alt={`${name} logo`}
```
Comment (lachlanjc) [inline src/components/EventCard.js]: We need these alt attributes

File: src/pages/404.js [hunk 2]
```diff
@@ -12,12 +12,7 @@ export default () => (
       <Text f={4} mt={2} mb={3} color="muted">
         We couldn’t find that page.
       </Text>
-      <LargeButton href="/">
-        Go Home{' '}
-        <span role="img" aria-label="house">
-          🏡
-        </span>
-      </LargeButton>
+      <LargeButton href="/">Go Home 🏡</LargeButton>
```
Comment (lachlanjc) [inline src/pages/404.js]: This is simpler but less accessible

File: src/components/EventCard.js (hunk 1 above)
Comment (maxwofford) [reply src/components/EventCard.js]: working on finding the cause of the off-by-one issue we've been having by rolling back a group of commits. I won't merge until I bring back all the useful improvements made in the past few weeks

Comment (lachlanjc) [reply src/components/EventCard.js]: Ah ok cool 👍

**Pull request closed without merging**